IOS/IOSPermissionsObject.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using UnityEngine;
+
+namespace DUCK.IOS
+{
+	public enum PermissionVariableType
+	{
+		String,
+		Integer,
+		Boolean,
+	}
+
+	[Serializable]
+	public class IOSPermission
+	{
+		public string key;
```

Comment:
Minor, but don't we normally make them private and expose via property, you did in the class below

Reply:
Agreed

Reply:
👍